test/tuple.jl

```diff
@@ -116,6 +116,10 @@ end
     # @test_throws ArgumentError size((1,2), 2)
 end
 
+t1 = (1, "a", "b")
```

Comment:
If these are only used in the test set you've added, they should be defined in that test set rather than at the top level.